frontend/components/settings/DeleteAccount.vue

```diff
@@ -0,0 +1,428 @@
+                },
+                footer: { class: 'h-0' },
+                closeButton: {
+                    class: 'justify-start w-full h-full items-center collapse',
+                },
+                mask: {
+                    class: 'sm:collapse bg-natural-50',
+                },
+            }"
+            @hide="close"
+        >
+            <template #header>
+                <button class="-ml-6 flex justify-center pr-4" @click="close">
+                    <span class="pi pi-angle-left text-3xl" />
+                </button>
+                <div class="font-nunito text-4xl font-semibold">
+                    <T key-name="dashboard.user.settings.delete" />
+                </div>
+            </template>
+            <div class="flex h-full flex-col pl-8">
+                <div
+                    class="-pt-4 w-11/12 overflow-hidden overflow-ellipsis text-sm text-natural-700 dark:text-natural-300"
+                >
+                    <T
+                        key-name="dashboard.user.settings.delete.account.description.part1"
+                    />
+                    <b class="text-text dark:text-natural-50">
+                        <T
+                            key-name="dashboard.user.settings.delete.account.description.part2"
+                    /></b>
+                    <T
+                        key-name="dashboard.user.settings.delete.account.description.part3"
+                    />
+                    <br />
+                    <br />
+                    <T
+                        key-name="dashboard.user.settings.delete.account.description.part4"
+                    />
+                </div>
+                <div class="flex items-center pl-6 pt-4">
+                    <div class="flex w-full flex-col items-center">
+                        <div
+                            class="mb-2 mr-10 flex w-full items-start text-sm dark:text-natural-50"
+                        >
+                            <T
+                                key-name="dashboard.user.settings.enter.current.password"
+                            />
+                        </div>
+                        <input
+                            id="passwordDelete"
+                            v-model="password"
+                            name="password"
+                            type="password"
+                            class="focus-ring-1 mb-3 mr-10 w-full rounded-md border-2 border-natural-400 bg-natural-100 py-1 pl-3 text-text placeholder:text-text hover:border-calypso-400 focus:border-calypso-400 focus:outline-none dark:border-natural-700 dark:bg-natural-800 dark:text-natural-50 dark:hover:border-calypso-400 dark:focus:border-calypso-400"
+                        />
+                        <span
+                            class="flex w-3/5 justify-start text-sm text-mahagony-600 dark:text-mahagony-300"
+                            >{{ errors.password }}</span
```

Comment:
Error message should be in english please 
![image](https://github.com/user-attachments/assets/04f87542-bf65-4f5f-8a45-7be23bf9a93a)